Marlin/Configuration.h

```diff
@@ -218,6 +218,8 @@ Here are some standard links for getting your machine calibrated:
 // so you shouldn't use it unless you are OK with PWM on your bed.  (see the comment on enabling PIDTEMPBED)
 #define MAX_BED_POWER 255 // limits duty cycle to bed; 255=full current
 
+//#define PID_BED_DEBUG // Sends debug data to the serial port.
```

Comment:
Add this line to the other Configuration.h files (in example_configurations) and we can commit this soon.